uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-tasks.js

```diff
@@ -21,5 +21,13 @@ module.exports = modal.extend({
   openCreateNewTaskModal: function () {
     this.chromy.click('.create-new-task');
     this.chromy.waitUntilVisible('#civihr-employee-portal-civi-tasks-form');
+  },
+
+  /**
+   * The page always gives false positives for some reason in Chromy, so we need
```

Comment:
Please add a `@NOTE` saying that this may still produce unstable results.

Reply:
mm but it doesn't, after this change i got consistent results